packages/react/src/VideoPlayer/VideoPlayer.features.stories.tsx

```diff
@@ -11,14 +12,85 @@ export default {
 
 export const WithPoster = () => (
   <VideoPlayer poster={posterImage} title="GitHub media player">
-    <VideoPlayer.Source src="/example.mp4" type="video/mp4" />
-    <VideoPlayer.Track src="/example.vtt" />
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
   </VideoPlayer>
 )
 
 export const WithoutBranding = () => (
-  <VideoPlayer branding={false} title="GitHub media player">
-    <VideoPlayer.Source src="/example.mp4" type="video/mp4" />
-    <VideoPlayer.Track src="/example.vtt" />
+  <VideoPlayer showBranding={false} title="GitHub media player">
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+  </VideoPlayer>
+)
+
+export const WithoutTitle = () => (
+  <VideoPlayer showTitle={false} title="GitHub media player">
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+  </VideoPlayer>
+)
+
+export const WithCustomControls = () => (
+  <VideoPlayer
+    title="GitHub media player"
+    renderControls={() => (
+      <ControlsBar>
+        <PlayPauseButton />
+        <SeekControl />
+      </ControlsBar>
+    )}
+  >
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+  </VideoPlayer>
+)
+
+export const HideControlsUntilPlaying = () => (
+  <VideoPlayer
+    title="GitHub media player"
+    renderControls={(controlsProps, {isPlaying}) => (
+      <Controls
+        style={{
+          opacity: isPlaying ? 1 : 0,
+          transition: 'opacity var(--brand-animation-duration-default) var(--brand-animation-easing-default)',
+        }}
```

Comment:
Yea absolutely. This example was mainly intended to show just how much control (bad pun, sorry) the implementor has over the controls. In practise I'd imagined that they'd toggle classes rather than use inline styles, but I wasn't sure how styling was set up in storybook.

Would you prefer that I added `hideControlsUnilPlaying` (I'll find a better name) as a prop, or that I update this example to remove the inline styling and instead toggle classes based on `isPlaying`?

Reply:
> I wasn't sure how styling was set up in storybook. 

You can create a new co-located file called `VideoPlayer.features.module.css` and import that into your Storybook file. It's then separate from the main stylesheet, and we use this approach for other components too.

> Would you prefer that I added hideControlsUnilPlaying (I'll find a better name) as a prop

👍 Yeah that would be great. It's a cool feature, so we should treat it as such by giving it it's own prop. Would you agree, or see issues doing that?

Reply:
Resolved in 4873e4713bad9c9ce1780008cf36c994d2f7053d